src/main/java/org/folio/codex/ContentType.java

```diff
@@ -0,0 +1,62 @@
+package org.folio.codex;
+
+import java.util.EnumMap;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.folio.rest.jaxrs.model.Package;
+
+public enum ContentType {
+
+  AGGREGATED_FULL_TEXT(Package.Type.AGGREGATED_FULL_TEXT, "aggregatedfulltext"),
+  ABSTRACT_AND_INDEX(Package.Type.ABSTRACT_AND_INDEX, "abstractandindex"),
+  E_BOOK(Package.Type.E_BOOK, "ebook"),
+  E_JOURNAL(Package.Type.E_JOURNAL, "ejournal"),
+  PRINT(Package.Type.PRINT, "print"),
+  UNKNOWN(Package.Type.UNKNOWN, "unknown"),
+  ONLINE_REFERENCE(Package.Type.ONLINE_REFERENCE, "onlinereference");
+
```

Comment:
I will update related raml so that is has consistent enum values (without spaces and casing) https://github.com/folio-org/raml/blob/raml1.0/schemas/codex/packageType.json

Following the naming convention here (it is Type in codex and ContentType in HoldingsIQ https://wiki.folio.org/pages/viewpage.action?pageId=1415393

Reply:
Actually it is ok to leave those enum values as is, this part just defines a mapping between Codex Package.Type enum and RM API contentType

Reply:
Understood -- it was also a good reminder that the enums defined in codex should probably have their format adjusted. I created a separate PR for this in the RAML project